src/debugger/ios/iOSPlatform.ts

```diff
@@ -24,6 +25,13 @@ export class IOSPlatform implements IAppPlatform {
     private simulatorTarget: string;
     private isSimulator: boolean;
 
+    // We should add the common iOS build/run erros we find to this list
+    private static RUN_IOS_FAILURE_PATTERNS: PatternToFailure = {
+        "No devices are booted": "Unable to launch iOS simulator. Try specifying a different target.",
+        "FBSOpenApplicationErrorDomain": "Unable to launch iOS simulator. Try specifying a different target." };

```

Comment:
Now that these are split up, we should give different errors... Although these both stem from not launching the correct emulator. The first case is when there are no emulators launched, while the second is when an incompatible emulator is launched.